src/CoreWf/CoreWf.csproj

```diff
@@ -25,15 +25,21 @@
     <AssemblyOriginatorKeyFile>..\..\CoreWF.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1">
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Scripting" Version="2.8.0-dev-62823-01" />
```

Comment:
This is from a MyGet source. Also I get a compatibility message, is there a way to avoid that? Is the Roslyn package available at that version on nuget somehow?

Reply:
Not that I'm aware of.

Reply:
@dmetzgar I'd like your input on adding that as a source before we merge this in. Is it OK to add this dependency? It is an MS package after all, just not (yet?) at that version on nuget.

@lbargaoanu Could this work with 
nuget install Microsoft.Net.Compilers   # Install C# and VB compilers
nuget install Microsoft.CodeAnalysis    # Install Language APIs and Services
from the https://github.com/dotnet/roslyn repo or do we need the custom one from myget?

Reply:
I tried adding all the nugets from Microsoft.CodeAnalysis.*, but couldn't find the Microsoft.CodeAnalysis.VisualBasic.Scripting - VisualBasicScript class. So it seems we require that MyGet feed for CoreWF.